src/EPR.SubsidiaryBulkUpload.Application/Models/CsvFileValidationConditions.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace EPR.SubsidiaryBulkUpload.Application.Models;
+
+public class CsvFileValidationConditions
+{
+    public const int MaxNoofColumnsAllowed = 6;
```

Comment:
Capitalisation of o on Of - should be MaxNoOfColumnsAllowed, or even MaxNumberOfColumnsAllowed

Reply:
changed